src/common/PersonDetailTile/index.js

```diff
@@ -0,0 +1,46 @@
+import {
+  DetailsTile,
+  NoImage,
+  Content,
+  Title,
+  Label,
+  Info,
+  Description,
+  Wrapper,
+  DetailsLine,
+} from "./styled";
+import { imagesAPIw400 } from "../../apiURL";
+import anonymous from "../../images/Picture.png";
+
+const PersonDetailTile = ({
+  poster,
+  name,
+  birthday,
+  birthplace,
+  biography,
+}) => {
+  return (
+    <Wrapper>
+      <DetailsTile>
+        <NoImage
+          src={poster ? `${imagesAPIw400}${poster}` : anonymous}
+          alt="name"
+        />
+        <Content>
+          <Title>{name}</Title>
+          <DetailsLine>
+            <Label>Date of birth:</Label>
+            <Info>{birthday}</Info>
+          </DetailsLine>
+          <DetailsLine>
+            <Label>Place of birth:</Label>
+            <Info>{birthplace}</Info>
```

Comment:
In the mobile version description ("Place of birth:") should be under brithplace.